.github/CODEOWNERS

```diff
@@ -0,0 +1,126 @@
+# Global owner for everything in this repo
+
+- @rt4914
+
+################################################################################
```

Comment:
Remove everything below this.

Reply:
removed. thanks

Reply:
This last line is still visible. Also line 3 there is some sort of dot visible. it should be star.
Overall file should be:
```
# Global owner for everything in this repo

* @rt4914

```

Reply:
Resolved. the issue. please check now. thanks. @rt4914

Reply:
Okay we will merge this, but if it creates an issue we will need to reopen the issue and fix this again.